portlets/src/main/webapp/vue-app/myContributions/components/MyContributions.vue

```diff
@@ -15,18 +15,18 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <gamification-overview-widget :see-all-url="achievementsURL" :loading="loading">
```

Comment:
Why not simply changing `gamification-overview-widget` to reuse widget-wrapper ?

Reply:
done